src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.task.TaskList;
+import duke.exception.DukeException;
+import duke.exception.TooManyTasksException;
+import duke.parser.DataParser;
+import duke.parser.DateParser;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Represents a Command to be executed during the running of the Duke Application.
+ */
+public class Command {
+
+    public boolean isExit;
```

Comment:
I was wondering if it would be better to set this Boolean as private/protected instead and accessed using a getter method?

Reply:
Hi Yi Yin!

I have noted this and applied a method checkIfExit() to access the value of private boolean value isExit.

Thanks!